### DISCUSSION TITLE
关于图片字号问题的询问

### DISCUSSION THREAD

**Discussion by HBULXY1997** [note286/xduts#112] [Q&A] [answered]
老哥你好，我在修改论文的时候发现，学校要求图片的字号为五号，但是我从visio中导入图片时，图片一般来说尺寸都比较大，所以会对图片进行调整，但是这样做会使得字体大小发生变化，请问老哥有什么建议关于图片字号大小调整。

**Comment by note286** [comment, marked as answer]
作图的时候就确定好图的尺寸（可参考 https://github.com/note286/xduts/discussions/40#discussioncomment-5237974 ），字号正常选5号即可。
